tests/units/DatabaseUpgradeTest.php

```diff
@@ -114,3 +145,36 @@ public function isMigrated(): bool
         return true;
     }
 }
+
+
+class PriorityEmergency implements Priority, MigratesTable
```

Comment:
this could be called `PriorityHighest`, I think emergency might be confusing, and we're not using it anywhere else, although we do have Default